src/command/cmd_defs.c

```diff
@@ -2102,6 +2104,7 @@ static const struct cmd_t command_defs[] = {
               { "set <account> otr <policy>", "Override global OTR policy for this account, see /otr." },
               { "set <account> pgpkeyid <pgpkeyid>", "Set the ID of the PGP key for this account, see /pgp." },
               { "set <account> startscript <script>", "Set the script to execute after connecting." },
+              { "set <account> client <client>", "Replace Jabber client's name to another one. For example, you can set it to PSI." },
```

Comment:
I think should be "Set XMPP client name according for discovery according to XEP-0092. Which only contacts that have presence subscription can query.

Reply:
Also you don't mention here that the version can be set as well and how. A example later on in `CMD_EXAMPLES` would be useful for this. This also then gets rid of the `For example, you can set it to PSI`.